src/System.Private.CoreLib/shared/System/Double.cs

```diff
@@ -40,59 +43,86 @@ public struct Double : IComparable, IFormattable, IComparable<Double>, IEquatabl
         public const double NaN = (double)0.0 / (double)0.0;
 
         // 0x8000000000000000 is exactly same as -0.0. We use this explicit definition to avoid the confusion between 0.0 and -0.0.
-        internal static double NegativeZero = Int64BitsToDouble(unchecked((long)0x8000000000000000));
+        internal static double NegativeZero = BitConverter.Int64BitsToDouble(unchecked((long)0x8000000000000000));
```

Comment:
Side note: This is internal and could be a constant. It could also just be a public member so that public users can "avoid confusion" as well (it would probably need to go through API review for that though).